src/Apps/Order/Routes/Shipping/FulfillmentDetailsForm.tsx

```diff
@@ -0,0 +1,830 @@
+                      onChosenAddress={(verifiedBy, chosenAddress) => {
+                        setAddressNeedsVerification(false)
+                        props.onSubmit(
+                          {
+                            ...values,
+                            attributes: {
+                              ...values.attributes,
+                              ...chosenAddress,
+                              addressVerifiedBy: verifiedBy,
+                            },
+                          },
+                          formikProps
+                        )
+                      }}
+                    />
+                  )}
+
+                  <GridColumns>
+                    <Column span={12}>
+                      <Input
+                        tabIndex={tabbableFormValue("new_address")}
+                        id="attributes.name"
+                        placeholder="Full name"
+                        title={"Full name"}
+                        autoCorrect="off"
+                        value={values.attributes.name}
+                        onChange={handleChange}
+                        error={
+                          touched.attributes?.name && errors.attributes?.name
+                        }
+                        data-testid="AddressForm_name"
+                      />
+                    </Column>
+
+                    <Column span={12}>
+                      <Text
+                        id="country-select"
+                        mb={0.5}
+                        variant="xs"
+                        color="black100"
+                      >
+                        Country
+                      </Text>
+                      <CountrySelect
+                        aria-labelledby="country-select"
+                        tabIndex={tabbableFormValue("new_address")}
+                        selected={values.attributes.country}
+                        onSelect={handleChange}
+                        disabled={
+                          !!lockShippingCountryTo &&
+                          lockShippingCountryTo !== "EU"
+                        }
+                        euShippingOnly={lockShippingCountryTo === "EU"}
+                        data-testid="AddressForm_country"
+                      />
+                      {lockShippingCountryTo && (
+                        <>
+                          <Spacer x={0.5} y={0.5} />
+                          <Text variant="xs" color="black60">
+                            {lockShippingCountryTo === "EU"
+                              ? "Continental Europe shipping only."
+                              : "Domestic shipping only."}
+                          </Text>
+                        </>
+                      )}
+                    </Column>
+                    <Column span={12}>
+                      {!autocomplete.loaded || autocomplete.enabled ? (
+                        <AutocompleteInput<AddressAutocompleteSuggestion>
```

Comment:
I've been keeping this input up to date with changes in its current in-development usage in AddressForm (eg #12898)